### PR TITLE
Gorkowski/issue5

### PR DESCRIPTION
**Pull request by Gorkowski**
I'm testing out that workflow, and adding that hype paragraph :)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,64 @@
-# particula
```
Comment (ngam) [inline README.md:1]: Sorry my comments were meant to be in the review here, but went as comments on the commits... My bad, sloppy!

File: README.md [hunk 2]
```diff
@@ -1 +1,64 @@
...
+
+### Codespaces
+
+The Codespaces feature is enabled for this organization. Click on "Code" (in green) on the right hand side and you can take it from there. Read more [here](https://github.com/features/codespaces). Basically a virtual machine that could be run in the browser or in VSCode.
+
+### Browser
+
+For light editing, try pressing the dot (`.`) on your keyboard, and this repository should turn from `github.com/uncscode/particula` to `github.dev/uncscode/particula` (changing from `.com` to `.dev`) --- now you could edit the files and commit! Read more [here](https://docs.github.com/en/codespaces/the-githubdev-web-based-editor). Related: [vscode.dev](https://vscode.dev).
+
+<!-- ## Experimental features
```
Comment (ngam) [inline README.md:58]: Delete this section :)

**Review by ngam — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (ngam) [reply README.md:58]: (the experimental features)

File: README.md (hunk 1 above)
Comment (Gorkowski) [reply README.md:1]: lol, I was like maybe I shouldn't read this until you finish your edits

**Review by ngam — APPROVED** [review]
Thanks @Gorkowski 🎆

**Pull request merged**